src/main.js

```diff
@@ -221,3 +237,32 @@ function createCover(imgSrc, title, descriptor1, descriptor2) {
   }
   return cover
 }
+
+function manuallyEditCover(element) {
+  element.contentEditable = true;
+  if(element.tagName === "IMG") {
+    let newImage = prompt("Please enter new URL for cover image:");
+    if(newImage) {
+      element.src = newImage;
+    }
+  }
+}
+
+function getNewRandomCoverItem(element) {
+  if(element.tagName === "IMG") {
+    var newRandomImage = getRandomImage();
+    coverImage.src = newRandomImage; 
+    currentCover.coverImg = newRandomImage;
+  } else if(element.tagName === "H2") {
+    var newRandomTitle = getRandomTitle();
+    coverTitle.innerText = newRandomTitle;
+    currentCover.title = newRandomTitle;
+  } else if(element.tagName === "H3") {
+    var desc1 = getRandomDesc();
+    var desc2 = getRandomDesc();
+    coverDescriptor1.innerText = desc1;
+    currentCover.tagline1 = desc1;
+    coverDescriptor2.innerText = desc2;
+    currentCover.tagline2 = desc2;
+  }
+}
```

Comment:
Looks like I just needed to mirror the JS object to what we were already setting the HTML to - I wonder if there is a more efficient way to couple those two things...